test/bg/user-script-registry.test.js

```diff
@@ -1,6 +1,6 @@
 describe('bg/user-script-registry', () => {
-  beforeEach(() => {
-    indexedDB.deleteDatabase('greasemonkey');
+  beforeEach(async () => {
+    await UserScriptRegistry._clearObjectStore();
```

Comment:
I believe this could be simplified to just `UserScriptRegistry._clearObjectStore` (since it now returns a promise).

Reply:
I wrote this as one part of beforeEach code but is it better to write `beforeEach(UserScriptRegistry._clearObjectStore);` better for code style?